tests/api/test_middleware.py

```diff
@@ -0,0 +1,24 @@
+from unittest.mock import Mock
+
+import pytest
+from django.urls import reverse
+
+from saleor.graphql.middleware import service_account_middleware
+
+
+@pytest.mark.parametrize("path, should_accept", [("api", True), ("home", False)])
```

Comment:
If one of test cases parameters is bool and we have condition based on it inside, there is strong sign that those are two separate test cases in reality.
This is not really huge scenario so maybe we could have to, well named tests for _api_ and _home_ ?